### DISCUSSION TITLE
Custom Domain is acting weirdly after moving from Netlify.

### DISCUSSION THREAD

**Discussion by imahmud1** [vercel/community#6456] [Help]
Hello everyone, hope you are doing well. My React+Vite project us primarily running on Netlify with a custom domain. Today i shifted to Vercel. I removed the custom domain and even the project from Netlify. Added vercel nameservers and configured domain in vercel.
Now the issue is i can access the site without any issues using the custom domain, even with VPN. But my friends who lives in Japan can not access this site. I tried using VPN where location was Singapore, usa and even Tokyo Japan it worked fine but not working properly for my friends and their friends who live in their prefecture. When i say not properly, they can sometimes access the site but a few mins later they can not.
I know Namesever change will take upto 48 hours to work globally and till now it's been 12+ hours since i added the domain. But they can access the site sometimes and sometimes they can't. 
When they try to access the site it shows a 404 served from Netlify not even vercel. 
What could be the issue?

**Comment by amyegan** [comment]
It could be that the nameserver change hasn't reached all global servers yet, as you suspected. It could also be local cache causing your  friend to see the wrong version of your site.

It's been a few hours since you originally posted this, so the problem may have already resolved. But if you're still having trouble, we have a guide with some troubleshooting tips that you can try: [vercel.com/docs/projects/domains/troubleshooting](https://vercel.com/docs/projects/domains/troubleshooting)
